### DISCUSSION TITLE
Grpc with caprover

### DISCUSSION THREAD

**Discussion by valensto** [firmanJS/express-template-typescript#13] [Q&A]
Hi, good job for this template! 

Just a question about the use of gRPC and Caprover, did you deploy on it? Do you have tips about it and what about tls and Caprover? If you can help me you'll save me a lot of time!

**Comment by firmanJS** [comment]
Hi, @valensto 

thank you for asking, for caprover I haven't tried it, grpc can try deploying in free hosted like Heroku
